util/design/gen-otp-img.py

```diff
@@ -26,22 +27,35 @@
 MEMORY_MEM_FILE = 'otp-img.BITWIDTH.vmem'
 
 
-def _override_seed(args, name, config):
+def _override_seed(args, seed_name, entropy_buffer_name, config):
     '''Override the seed key in config with value specified in args'''
-    arg_seed = getattr(args, name)
-    # An override seed of 0 will not trigger the override, which is intended, as
-    # the OTP-generation Bazel rule sets the default seed values to 0.
-    if arg_seed:
-        log.warning('Commandline override of {} with {}.'.format(
-            name, arg_seed))
-        config['seed'] = arg_seed
-    # Otherwise, we either take it from the .hjson if present, or
-    # randomly generate a new seed if not.
+    arg_seed = getattr(args, seed_name)
+    arg_entropy_buffer = None
+    if entropy_buffer_name:
+        arg_entropy_buffer = getattr(args, entropy_buffer_name)
+
+    if arg_entropy_buffer:
+        if arg_seed:
+            log.error(
+                "{} 'entropy_buffer' option cannot be used with {} 'seed' option"
+                .format(entropy_buffer_name, seed_name))
+            exit(1)
+        else:
+            config['entropy_buffer'] = arg_entropy_buffer
     else:
-        new_seed = random.getrandbits(64)
-        if config.setdefault('seed', new_seed) == new_seed:
-            log.warning('No {} specified, setting to {}.'.format(
-                name, new_seed))
+        # An override seed of 0 will not trigger the override, which is intended, as
+        # the OTP-generation Bazel rule sets the default seed values to 0.
+        if arg_seed:
+            log.warning('Commandline override of {} with {}.'.format(
+                seed_name, arg_seed))
+            config['seed'] = arg_seed
+        # Otherwise, we either take it from the .hjson if present, or
+        # randomly generate a new seed if not.
```

Comment:
Is this intended or did you want to align with the behavior used in all other places (i.e., error out)?

Reply:
same here, am going to do in a follow up, since I will need to modify several OTP HJSON files.